config/routes.rb

```diff
@@ -6,6 +6,7 @@
   namespace :dor do
     # TODO: Deprecate GET when caml POST can be implemented
     match 'reindex/:pid', action: :reindex, via: %i[get post put]
+    match 'reindex_from_cocina/:pid', action: :reindex_from_cocina, via: %i[get post put]
```

Comment:
`get` and `post` should not be options here.  This is a PUT: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT

Reply:
d'oh, yup, fixing...